apps/etl-func/src/func.ts

```diff
@@ -34,6 +57,27 @@ const main = async (config: Config) => {
     methods: ["GET"],
     route: "health",
   });
+
+  app.cosmosDB("messagesCosmosDBTrigger", {
+    connection: "COSMOS",
+    containerName: config.cosmos.messagesContainerName,
+    createLeaseCollectionIfNotExists: false,
+    databaseName: config.cosmos.databaseName,
+    handler: messagesIngestion(messageAdapter, PDVTokenizer, producer),
+    leaseContainerName: `${config.cosmos.messagesContainerName}-ingestion-lease`,
```

Comment:
We should hardcode the name here in my opinion, if the container name changes we do not want to re-ingest.